### PR TITLE
added some resources

### PR DESCRIPTION
**Pull request by Adesh-123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +159,11 @@ If the PR is not according to the guidelines then it will not be accepted.
 5. [Step-by-step guide to learn CSS](https://htmldog.com/guides/css/).
 6. [CSS References](https://htmldog.com/references/css/).
 7. [Grid Garden](https://cssgridgarden.com/)
+8. [CSS Full course](https://www.youtube.com/watch?v=Edsxf_NBFrw&t=17808s)
+
+### SASS <a name="sass"></a>
+
+1.[SASS](https://www.youtube.com/watch?v=_a5j7KoflTs&t=774s)
```
Comment (aaryan2134) [inline README.md:166]: Please remove space between the numbering and the title
Also, the link points to a specific time in the video. Please remove that or provide some explanation on that

File: README.md [hunk 2]
```diff
@@ -193,6 +200,10 @@ If the PR is not according to the guidelines then it will not be accepted.
 5.  [Learn React](https://scrimba.com/learn/learnreact).
 6.  checkout courses on udemy .
 
+### Node.js <a name="node.js"></a>
+
+1.[Node.js full course](https://www.youtube.com/watch?v=VrQgmNY96wo)
```
Comment (aaryan2134) [inline README.md:205]: Please remove space between the numbering and the title

**Review by aaryan2134 — CHANGES_REQUESTED** [review]
Please make these changes and add NodeJS and SAAS in the index of the readme as well

**Comment by Adesh-123** [comment]
resolved the issue

**Comment by Adesh-123** [comment]
issue fixed

**Comment by aaryan2134** [comment]
Good Work!
Thank you for your contribution

**Pull request merged**